client/layout/universal-navbar-footer/index.tsx

```diff
@@ -171,11 +174,14 @@ const UniversalNavbarFooter = () => {
 										{ translate( 'Daily Webinars' ) }
 									</a>
 								</li>
-								<li>
-									<a href={ localizeUrl( 'https://wordpress.com/courses/' ) } target="_self">
-										{ translate( 'WordPress Courses' ) }
-									</a>
-								</li>
+								{ ( localeFromPath === 'en' ||
```

Comment:
I assume you switch to using `localeFromPath`, because `getLocaleSlug()` always returns the default `en` locale slug on the SSR.

That's happens because `getLocaleSlug()` reads the locale slug from the [global i18n-calypso instance](https://github.com/Automattic/wp-calypso/blob/trunk/packages/i18n-calypso/src/index.js#L18), whereas for the plugins and themes sections, we create a [private i18n-calypso instance](https://github.com/Automattic/wp-calypso/blob/trunk/client/controller/ssr-setup-locale.js#L14).

If you want to get the actual locale slug from the private i18n-calypso instance, you can use either:

```js
const { localeSlug } = useTranslate();
```

or:

```js
import { useLocale } from '@automattic/i18n-utils';
```

Both approaches are using the same source for the locale slug (i.e. `i18n.getLocaleSlug()` [#1](https://github.com/Automattic/wp-calypso/blob/trunk/packages/i18n-calypso/src/use-translate.js#L6) [#2](https://github.com/Automattic/wp-calypso/blob/trunk/client/components/calypso-i18n-provider/index.tsx#L13)), so it shouldn't really matter which one you use.

As for the conditional rendering check - it could probably be replaced by [isDefaultLocale](https://github.com/Automattic/wp-calypso/blob/trunk/packages/i18n-utils/src/utils.ts#L28) or [useIsEnglishLocale](https://github.com/Automattic/wp-calypso/blob/trunk/packages/i18n-utils/src/locale-context.tsx#L118).

That being said, the use of `localeFromPath` seems to be redundant, unless there's another reason for using it that I'm missing.

Reply:
@yuliyan

The reason for using `localeFromePath` is we need to determine what language is the given page translated in. In english Locale, both `/themes` and `/es/themes` returns `true` for the method `useIsEnglishLocale` and returns `en` for the method `useTranslate` - we do not want this.

Basically, for GoogleBot and users who land on the page /es/themes (and others in non English language) the /learn/ link should not be displayed in the menu as /learn/ is not available in any non-english language.

Reply:
@dkd903, if `useIsEnglishLocale` returns `true` in all cases on your local testing, it could be due to missing translation files. Could you try booting up Calypso with `BUILD_TRANSLATION_CHUNKS=true yarn start` to ensure translation files are downloaded?

I've just tested on my environment and I'm seeing the expected results.

Reply:
@yuliyan - that actually did the trick. I have updated the PR and thank you so much for your help and guidance here!